internal/vulnstore/enrichment.go

```diff
@@ -0,0 +1,22 @@
+package vulnstore
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+// Enricher is an interface exporting the necessary methods
+// for storing and querying Enrichments.
+type EnrichmentUpdater interface {
+	// UpdateEnrichments creates a new EnrichmentUpdateOperation, inserts the provided
+	// EnrichmentRecord(s), and ensures enrichments from previous updates are not
+	// queries by clients.
+	UpdateEnrichments(ctx context.Context, mime string, fingerprint driver.Fingerprint, enrichments []driver.EnrichmentRecord) (uuid.UUID, error)
+}
+
+type Enrichment interface {
+	GetEnrichment(ctx context.Context, mime string, tags []string) ([]driver.EnrichmentRecord, error)
```

Comment:
The `Vulnerability` equivalent is just `Get`, what are your thoughts on making them the same?

Reply:
Not opposed, just probably another PR for another day.